protos/sources/vega/events/v1/events.proto

```diff
@@ -96,6 +98,13 @@ message AMM {
     // Leverage at lower bounds.
     optional string leverage_at_lower_bound = 5;
   }
+
+  message Curve {
+    // Unit liquidity on the given AMM curve.
```

Comment:
I can't figure out what this string is telling me - is it a unit of liquidity, the entirety of the liquidity, or maybe a third option...